frontend/src/components/Timer/Timer.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+
+class Timer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      seconds: "00",
+      value: props.time,
+      loading: true
+    };
+  }
+
+  componentDidMount() {
+    this.startCountDown();
+  }
+
+  tick = () => {
+    let min = Math.floor(this.secondsRemaining / 60);
+    let sec = this.secondsRemaining - min * 60;
+
+    this.setState({
+      value: min,
+      seconds: sec
+    });
+
```

Comment:
these are great spots to use the ternary trick and have it all on one line 

setState(sec < 10 ? seconds : "0" + seconds)